### PR TITLE
feat: APP-2915 - Implement Illustrations from ODS

### PR DESCRIPTION
**Pull request by thekidnamedkd**
## Description

Updates all straight forward usages of the illustration components for Human and Object that are not deeply nested in various forms of EmptyState implementation.

Task: [APP-2915](https://aragonassociation.atlassian.net/browse/APP-2915)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.

[APP-2915]: https://aragonassociation.atlassian.net/browse/APP-2915?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/finance.tsx [hunk 1]
```diff
@@ -78,16 +78,12 @@ export const Finance: React.FC = () => {
           Illustration={
             <div className="flex">
               <IllustrationHuman
```
Comment (Fabricevladimir) [inline src/pages/finance.tsx]: For whatever reason, the illustration isn't showing here.

**Review by Fabricevladimir — COMMENTED** [review]
Thanks, nice work! 
I did notice that we have `illuHuman` & `illuObject` refrences in `src->components->stateEmpty`. Is the intention to remove this when the empty state is replaced?

**Comment by thekidnamedkd** [comment]
> Thanks, nice work! I did notice that we have `illuHuman` & `illuObject` refrences in `src->components->stateEmpty`. Is the intention to remove this when the empty state is replaced?

Yeah that's the intention since EmptyState is coming up next and there's a lot of haranguing to be done per usage. @Fabricevladimir

**Comment by thekidnamedkd** [comment]
> > Thanks, nice work! I did notice that we have `illuHuman` & `illuObject` refrences in `src->components->stateEmpty`. Is the intention to remove this when the empty state is replaced?
> 
> Yeah that's the intention since EmptyState is coming up next and there's a lot of haranguing to be done per usage. @Fabricevladimir

Just kidding @Fabricevladimir -- I just made the changes cuz I was about to do them in the IconWork anyhow, so circled back here.

File: src/components/stateEmpty/index.tsx [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
-  IlluHumanProps,
-  IlluObject,
-  IlluObjectProps,
-  IllustrationHuman,
-} from '@aragon/ods-old';
+  Button,
+  IIllustrationHumanProps,
+  IIllustrationObjectProps,
+} from '@aragon/ods';
+import {IllustrationObject, IllustrationHuman} from '@aragon/ods';
```
Comment (Fabricevladimir) [inline src/components/stateEmpty/index.tsx]: Please combine these two before merging! Thanks

**Review by Fabricevladimir — APPROVED** [review]

**Pull request merged**